docs/proposals/deployment-insanity.md

```diff
@@ -0,0 +1,29 @@
+Dropping into a doc so we have a place to argue.
+
+We want to enabling deployments(Ds) and someday we want feature parity with DCs and then we'll likely retire DCs.
+
+This means that we need to make sure that no D or DC have conflicting names.  The simplest way to 
+do this is to have the two resources cohabitate (live in the same directory in etcd).  However,
+they are different than our other migrating/cohabitating resources like replicationcontrollers and 
+replicasets.  RC and RS controllers are close enough to parity that we're going to be able to run
+a single controller to manage both.  D and DC controllers are not close enough to do that.
```

Comment:
As a 1.3 objective or 1.4?  Going partway for 1.3 doesn't preclude making it work more completely for 1.4.